ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly-4.10-upgrade-from-stable-4.9.yaml

```diff
@@ -69,6 +69,19 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test-disconnected
     workflow: cucushift-installer-rehearse-aws-c2s-ipi-disconnected-private
+- as: aws-c2s-ipi-disconnected-private-fips-cert-rotation-p2-f14
```

Comment:
I think f14 can not met our test requirements at the moment.

Reply:
I'll use the PR and API to trigger everything for 4.10 for the push on 4.10 is very intense.